packages/contracts/deployments/mainnet/OVM_ExecutionManager.json

```diff
@@ -670,38 +671,38 @@
       "type": "function"
     }
   ],
-  "transactionHash": "0x5e986b048e3fc50a6186604f8048c310d4bcf7790dc790a41bf6eb337c8d4d48",
+  "transactionHash": "0xde8005c84c3179ad305e8a09dc97d3147344df3658bb76d7570a7721066f7eee",
   "receipt": {
     "to": null,
-    "from": "0x77C4c8f491e5ff22a2C0E18b005c3BA0Fe36c1Bc",
-    "contractAddress": "0xEd93C5c21c502bB52b4D77240fA9a5d38472304d",
-    "transactionIndex": 41,
-    "gasUsed": "3535831",
+    "from": "0xd033f09cB85621F98F7A84C64dB381ac16Eff818",
+    "contractAddress": "0x3f5FA555c434b49D946042955013966Fd108DaC3",
+    "transactionIndex": 194,
+    "gasUsed": "2944657",
     "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
-    "blockHash": "0x79facf003b0e15ebca3063dade3622d78db0537262b6165e260ed323fb158c7f",
-    "transactionHash": "0x5e986b048e3fc50a6186604f8048c310d4bcf7790dc790a41bf6eb337c8d4d48",
+    "blockHash": "0x9dd55e729aebcd0316698d6b7efebeef7ee1cc12d129ec22595107be4ee9c554",
+    "transactionHash": "0xde8005c84c3179ad305e8a09dc97d3147344df3658bb76d7570a7721066f7eee",
     "logs": [],
-    "blockNumber": 12207854,
-    "cumulativeGasUsed": "7070914",
+    "blockNumber": 12410837,
+    "cumulativeGasUsed": "10740618",
     "status": 1,
     "byzantium": true
   },
   "args": [
-    "0xd3EeD86464Ff13B4BFD81a3bB1e753b7ceBA3A39",
+    "0x668E5b997b9aE88a56cd40409119d4Db9e6d752E",
     {
-      "minTransactionGasLimit": 50000,
+      "minTransactionGasLimit": 100000,
```

Comment:
Can we double check that there are no problems with 100k min

Reply:
@ben-chain check me on this one, but I think that 100k should be fine considering we have this check in the deposits: https://github.com/ethereum-optimism/optimism/blob/52d9e6b73362a203bdae3fa0182bd6e7c9693b13/packages/contracts/contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol#L41

Reply:
Yeah, these should be combined and always kept the same.  I agree that this is OK due to the current require that Karl pointed out.  One note is it [will make things OOG more often](https://github.com/ethereum-optimism/optimism/blob/52d9e6b73362a203bdae3fa0182bd6e7c9693b13/packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol#L209), though.